src/events/message_delete.rs

```diff
@@ -68,19 +68,50 @@ pub async fn message_delete(
         .map(|entry| entry.user_id)
         .and_then(|deletor| audit_logs.users.iter().find(|usr| usr.id == deletor));
 
+    let image = if let Some(_image) = msg
+        .attachments
+        .iter()
+        .filter(|a| a.height.is_some() && a.height.unwrap() > 0)
+        .next()
+    {
+        let image_path = std::path::Path::new("cache").join(msg.id.to_string());
+        if image_path.exists() {
+            Some(tokio::fs::File::open(image_path).await.unwrap())
```

Comment:
`.unwrap()` :((((  there may be other errors than just "file doesn't exist", so unwrapping here shouldn't be necessary - you can replace this with
```suggestion
            tokio::fs::File::open(image_path).await.ok()
```
for example - this will just return `None` if any error occurs. better logging/error reporting would of course be preferable, but in this case it's rather unlikely for anything to go wrong so it should be fine